src/netex_france/exporter.rs

```diff
@@ -183,6 +186,20 @@ impl Exporter<'_> {
         Ok(frames)
     }
 
+    // Returns a list of 'ServiceFrame' each containing a 'Network'
```

Comment:
// Returns a list of 'Line'

Reply:
Is it so obvious I used copy/paste to often? :roll_eyes:

Reply:
Fixed in 8d0bd72034c3f87d460c28ccd9dbcbace0216c5c.